target/linux/ath79/image/common-yuncore.mk

```diff
@@ -0,0 +1,11 @@
+define Build/yuncore-tftp-header-ap147
+	( \
+		echo -n -e "YUNCOREsetenv bootcmd \"bootm 0x9f050000 || bootm 0x9fe80000\" &&" \
+			"saveenv && erase 0x9f050000 +0xfa0000 && cp.b 0x800600c0 0x9f050000 0xfa0000" | \
+		dd bs=192 count=1 conv=sync; \
+		  dd if=$@; \
+	) > $@.new
+  mv $@.new $@
```

Comment:
Spaces for indent by intention?

Reply:
Nope. Did the changes with Github UI which replaces tabs with spaces. Going to fix it.